app/src/main/java/ru/netology/fmhandroid/api/PatientApiService.kt

```diff
@@ -0,0 +1,58 @@
+package ru.netology.fmhandroid.api
+
+import androidx.viewbinding.BuildConfig
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.*
+import ru.netology.fmhandroid.BuildConfig.BASE_URL
+import ru.netology.fmhandroid.dto.Admission
+import ru.netology.fmhandroid.dto.Note
+import ru.netology.fmhandroid.dto.Patient
+import java.util.concurrent.TimeUnit
+
+
+private val loggin = HttpLoggingInterceptor().apply {
+    if (BuildConfig.DEBUG) {
+        level = HttpLoggingInterceptor.Level.BODY
+    }
+}
+
+private val okhttp = OkHttpClient.Builder()
+    .addInterceptor(loggin)
+    .connectTimeout(30, TimeUnit.SECONDS)
+    .build()
+
+private val retrofit = Retrofit.Builder()
+    .addConverterFactory(GsonConverterFactory.create())
+    .baseUrl(BASE_URL)
+    .client(okhttp)
+    .build()
+
+interface PatientApiService {
+    @GET("patient?offset=0&limit=30&show_active=false")
+    suspend fun getAllPatients(): Response<List<Patient>>
+
+    @GET("patient/{id}")
+    suspend fun getPatientById(@Path("id") id: Int): Response<Patient>
+
+    @GET("patient/{id}/admission")
+    suspend fun getPatientAdmissions(@Path("id") id: Int): Response<List<Admission>>
+
+    @GET("patient/{id}/note")
+    suspend fun getPatientNotes(@Path("id") id: Int): Response<List<Note>>
+
+    @POST("patient")
+    suspend fun savePatient(@Body patient: Patient): Response<Patient>
+
+    @PATCH("patient")
+    suspend fun editPatient(@Body patient: Patient): Response<Patient>
+
+    object PatientApi {
```

Comment:
```suggestion
    companion object {
```
И вуаля - тебе не надо запоминать ещё одно имя класса (и придумывать его, что ещё страшнее 🙀)

Reply:
Хм. И как я сам не догадался, отличный вариант, везде где мы обращаемся к Api Service теперь код меньше и красивее)))